lxd/instances_post.go

```diff
@@ -404,6 +405,181 @@ func createFromMigration(s *state.State, r *http.Request, projectName string, pr
 	return operations.OperationResponse(op)
 }
 
+// createFromConversion uploads the image to the server's backup directory, converts it into the
+// the raw format, and imports the LXD instance using the converted image.
+func createFromConversion(s *state.State, r *http.Request, projectName string, profiles []api.Profile, req *api.InstancesPost) response.Response {
+	if s.DB.Cluster.LocalNodeIsEvacuated() && r.Context().Value(request.CtxProtocol) != "cluster" {
```

Comment:
Is there a situation where the CtxProtocol is "cluster" and we would still allow a new instance to be created on the local node? If not do we need the 2nd condition here?

Reply:
Couldn't find any situation, therefore, removed a second condition.